app/src/main/java/com/example/ncov19traking/ui/graphs/GraphsFragment.kt

```diff
@@ -19,7 +21,7 @@ import com.github.mikephil.charting.interfaces.datasets.ILineDataSet
 
 class GraphsFragment : Fragment() {
 
-    private val notificationsViewModel by lazy {
+    private val graphsViewModel by lazy {
         ViewModelProvider(this).get(GraphsViewModel::class.java)
```

Comment:
I think kotlin extensions (KTX) has an extension function to do this more cleanly, something like 
`val viewModel by viewModels<MyViewModel>()`
 something like that, for now you don't need to change this if you don't want to but you should check [KTX](https://developer.android.com/kotlin/ktx#fragment) just so you know they exists hahaha.

Reply:
Good to know, fixed